eos-tech-support/eos-list-users

```diff
@@ -1,27 +1,41 @@
 #!/bin/bash -e
 
-# Lists all users (i.e., all home directories on /dev/sda2)
-
-DEVICE=/dev/sda2
-MOUNT=/mnt/sda2
+# Lists all users (i.e., all home directories on all ostree devices)
 
 USERID=$(id -u)
 if [ "$USERID" != "0" ]; then
     echo "Program requires superuser privileges"
     exit 1
 fi
 
+MOUNT=`mktemp -d`
+
 cleanup() {
     set +e
-    umount $MOUNT
+    if mount | grep $MOUNT; then
```

Comment:
I tried that first, but it still spits out an error message (and because of the loop, the normal case is that the device has been unmounted upon a clean exit)